modules/migration-mtc-cr-manifests.adoc

```diff
@@ -47,16 +47,16 @@ kind: DirectImageStreamMigration
 metadata:
   labels:
     controller-tools.k8s.io: "1.0"
-  name: directimagestreammigration_name
+  name: <directimagestreammigration>
```

Comment:
Consider putting an underscore between the words, like you have for image_stream below.